app/logic/Contacts/OWA/OWAAddressbook.ts

```diff
@@ -32,28 +32,14 @@ export class OWAAddressbook extends Addressbook {
   }
 
   async listContactsSlow() {
-    let response = await this.account.callOWA(new OWAGetPeopleFiltersRequest());
-    let contacts = response.find(filter => !filter.IsReadOnly);
-    if (!contacts) {
-      this.persons.clear();
-      this.groups.clear();
```

Comment:
The old code assumes that it starts with an empty address book. Did you verify that this is not a problem in the new code?